nmdc_runtime/site/changesheets/base.py

```diff
@@ -0,0 +1,81 @@
+# nmdc_runtime/site/changesheets/base.py
+"""
+base.py: Provides data classes for creating changesheets for NMDC database objects.
+"""
+
+import logging
+import time
+from dataclasses import dataclass
+from pathlib import Path
+import requests
+from typing import Any, ClassVar, Dict, TypeAlias
+
+from nmdc_runtime.site.resources import RuntimeApiUserClient
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s %(levelname)s %(" "message)s"
+)
+
+JSON_OBJECT: TypeAlias = Dict[str, Any]
+CHANGESHEETS_DIR = Path(__file__).parent.absolute().joinpath("changesheets_output")
+
+
+@dataclass
+class ChangesheetLineItem:
+    """
+    A line item in a changesheet
+    """
+
+    id: str
+    action: str
+    attribute: str
+    value: str
+
+    @property
+    def line(self) -> str:
+        return f"{self.id}\t{self.action}\t{self.attribute}\t{self.value}"
+
+
+@dataclass
+class Changesheet:
+    """
+    A changesheet
+    """
+
+    name: str
+    line_items: list = None
```

Comment:
I think `None` is not a valid value of type `list`.

You could either change the value to `[]`, or change the type to either `list | None` ([docs](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#x-y-syntax-for-unions)) or `Optional[list]` ([docs](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type)). `Optional` comes from `typing`; as in: `from typing import Optional`

Reply:
It's actually `line_items: list = field(default_factory=list)`